### ISSUE TITLE
How to fine-tune a pre-trained model to suit one’s own needs?

### ISSUE THREAD

**Issue by Ggggfgjjjk** [AILab-CVC/YOLO-World#6]
Hello author, excellent work! I would like to ask how to fine-tune a model to suit my own needs using a small dataset of my own? The main purpose is to use it for annotating a large amount of data.Thank you!

**Comment by wondervictor** [comment]
Hi @Ggggfgjjjk, thanks for your interest in YOLO-World! You can modify the pre-training config to support your own datasets and load the pre-trained models. We suggest that the learning rate is set to 2e-4 and batch size is set to 16 for each device (8 GPUs are enough for fine-tuning). BTW, we will release the code/configs for fine-tuning YOLO-World on COCO and LVIS soon, and please stay tuned!

**Comment by wondervictor** [comment]
Hi @Ggggfgjjjk, we provide a simple guide for fine-tuning YOLO-World in [docs/fine-tuning](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/finetuning.md). Hope it helps!

**Comment by wondervictor** [comment]
Thanks for your interest. If you have any questions about YOLO-World in the future, you're welcome to open a new issue.
